app/routes/content.js

```diff
@@ -104,7 +104,8 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated') || null).toISOString()]
+                ['article:published_time', new Date(preprint.get('datePublished')).toISOString()],
```

Comment:
Interesting. Under what circumstances can a preprint be used with this widget but not have a creation date?

If you create a date object without having a date, it seems this would default to Jan 1 1970, which is probably not an accurate choice of date. Is that the desired behavior?